site/docs/id/README.md

```diff
@@ -86,5 +85,13 @@ Berhasil! :tada:
 
 ---
 
-grammY mendukung API Bot Telegram versi 6.6 yang [dirilis](https://core.telegram.org/bots/api#march-9-2023) pada tanggal 9 Maret 2023.
+<ClientOnly>
+  <ThankYou :s="['Terima kasih kepada', 'Someone', 'karena telah menjadi kontributor grammY.']" />
```

Comment:
We can shorten it to 'Terima kasih Someone telah menjadi kontributor GrammY.' It sounds more natural to me.

![2](https://user-images.githubusercontent.com/74030149/224600889-bc277458-2ca0-408e-a4b9-e7fa9748bb7d.png)

```suggestion
  <ThankYou :s="['Terima kasih ', 'Someone', ' telah menjadi kontributor grammY.']" />
```

Reply:
Ok.